assets/src/stories-editor/components/higher-order/with-call-to-action-validation.js

```diff
@@ -1,61 +1,69 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
 import { getBlockType } from '@wordpress/blocks';
 import { Button } from '@wordpress/components';
-import { withSelect, withDispatch } from '@wordpress/data';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { Warning } from '@wordpress/block-editor';
-import { createHigherOrderComponent, compose } from '@wordpress/compose';
+import { createHigherOrderComponent } from '@wordpress/compose';
 import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { isCTABlock } from '../../helpers';
 
-const enhance = compose(
-	/**
```

Comment:
This comment has been removed? Why? I dont like removing docs where possible.

Reply:
It is not gone, it just has been moved to the bottom and slightly reworded.

Reply:
:+1: